docs/glossary.mdx

```diff
@@ -468,6 +480,6 @@ A command line tool that zkApp developers use to scaffold and deploy smart contr
 
 ### zk-SNARK
 
-A type of zero-knowledge proof. zk-SNARK is the acronym for zero knowledge succinct non-interactive argument of knowledge. Specific properties of interest in Mina's implementation of SNARKs are succinctness and non-interactivity that allow for any node to quickly verify the state of the network.
+A recursive zero knowledge proof. zk-SNARK is the acronym for zero knowledge succinct non-interactive argument of knowledge. Specific properties of interest in Mina's implementation of SNARKs are succinctness and non-interactivity that allow for any node to quickly verify the state of the network.
```

Comment:
There are some kinds of recursive zero knowledge proofs that are generally not referred to as SNARKs (the SNARK name is a bit confusing and isn't especially well-defined).

Reply:
good call, I removed "recursive" 

I got the recursive bit from https://minaprotocol.com/blog/what-are-zk-snarks 

We have a lot of clarity to unmuddy